### PR TITLE
Added support for RGB Matrix FeatherWing

### LINKED ISSUE #22: Add Feather + Matrix FeatherWing init code or example
A user on twitter asked about using this library for the RGB Matrix FeatherWing. If they just wanted to use the graphics layer, this might be handy. So adding either some initialization code or an example might be a good idea.

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #22. This also reduces the number of decision branches and informs Metro Users they need to pass in alt_addr_pins if using a 16x32 matrix.

Tested with:
 * MatrixPortal M4
 * Metro M4 Airlift Lite with RGB Matrix Shield
 * Feather M4 with RGB Matrix FeatherWing

### CODE REVIEW

**Comment by makermelissa** [comment]
I forgot to mention, this is useful if somebody wants to have it automatically configure for their setup regardless of network connectivity.

File: adafruit_matrixportal/matrix.py [hunk 1]
```diff
@@ -70,23 +66,33 @@ def __init__(self, *, width=64, height=32, bit_depth=2, alt_addr_pins=None):
...
-        if hasattr(board, "MTX_CLK"):
             latch_pin = board.MTX_LAT
-        else:
-            latch_pin = board.D10
-
-        if hasattr(board, "MTX_OE"):
             oe_pin = board.MTX_OE
-        else:
+        elif hasattr(board, "D7"):
+            # Metro/Grand Central Style Board
```
Comment (ladyada) [inline adafruit_matrixportal/matrix.py:82]: i think maybe better to look in os.uname?

**Review by ladyada — CHANGES_REQUESTED** [review]

File: adafruit_matrixportal/matrix.py (hunk 1 above)
Comment (makermelissa) [reply adafruit_matrixportal/matrix.py:82]: Cool, I looked for something like that but didn't see it the first time. Thanks.

**Comment by makermelissa** [comment]
Updated and tested on all 3 boards again.

**Review by ladyada — APPROVED** [review]

**Pull request merged**